floss/language/identify.py

```diff
@@ -131,15 +131,30 @@ def is_go_bin(pe: pefile.PE) -> bool:
                     return True
 
     # if not found, the magic bytes may have been patched, search for common Go functions present in all Go samples including obfuscated files
-    for go_function in go_functions:
-        for section in pe.sections:
+    # look for the .rdata section first
+    for section in pe.sections:
+        try:
+            section_name = section.Name.partition(b"\x00")[0].decode("utf-8")
+        except UnicodeDecodeError:
+            continue
+        if ".rdata" == section_name:
```

Comment:
maybe reuse this?
https://github.com/mandiant/flare-floss/blob/152316f79b075115790b9484c5196a2429000598/floss/language/rust/extract.py#L20-L25

could move this to utils most likely

Reply:
Could move this then to https://github.com/mandiant/flare-floss/blob/master/floss/language/utils.py